### PR TITLE
tidy: journal

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: src/vsr/journal.zig [hunk 1]
```diff
@@ -2089,7 +2110,7 @@ pub fn JournalType(comptime Replica: type, comptime Storage: type) type {
 
             // TODO The compiler should not need this align cast as the type of `headers_iops`
```
Comment (chaitanyabhandari) [inline src/vsr/journal.zig]: Shall we remove this todo now that the @alignCast is removed?

Comment (matklad) [reply src/vsr/journal.zig]: Good catch, thnks!

Comment (matklad) [reply src/vsr/journal.zig]: fixed&rebased!

**Comment by matklad** [comment]
This is the last one!

**Review by chaitanyabhandari — APPROVED** [review]

**Comment by batiati** [comment]
> This is the last one!

Previous ones:
#2097
#2098
#2100
#2092
#2094
#2089
#2073
#2087
#2086
#2083
#2085

**Pull request merged**